rpms/SPECS/wazuh-manager.spec

```diff
@@ -842,6 +842,8 @@ rm -fr %{buildroot}
 %attr(750, root, wazuh) %{_localstatedir}/wodles/gcloud/*
 
 %changelog
+* Mon May 22 2023 support <info@wazuh.com> - 4.4.3
```

Comment:
Wrong date

Reply:
Why not to do this change here explained in https://github.com/wazuh/wazuh-packages/pull/2216#issuecomment-1560637572